corehq/apps/dump_reload/sql/filters.py

```diff
@@ -21,7 +24,7 @@ class SimpleFilter(DomainFilter):
     def __init__(self, filter_kwarg):
         self.filter_kwarg = filter_kwarg
 
-    def get_filters(self, domain_name):
+    def get_filters(self, domain_name, db_alias):
```

Comment:
Why does `db_alias` not have a default here while it does for some of the other filters?